rosidl_typesupport_freertps_cpp/cmake/rosidl_typesupport_freertps_cpp_generate_interfaces.cmake

```diff
@@ -200,43 +112,22 @@ configure_file(
 list(APPEND _generated_msg_files "${_visibility_control_file}")
 
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
-  #if(NOT "${_generated_msg_files}${_generated_external_msg_files} " STREQUAL " ")
-  if(NOT "${_generated_msg_files} " STREQUAL " ")
-    #  FILES ${_generated_msg_files} ${_generated_external_msg_files}
-    install(
-      FILES ${_generated_msg_files}
-      DESTINATION "include/${PROJECT_NAME}/msg/dds_freertps"
-    )
-  endif()
-  #if(NOT "${_generated_srv_files}${_generated_external_srv_files} " STREQUAL " ")
-  if(NOT "${_generated_srv_files} " STREQUAL " ")
-    #  FILES ${_generated_srv_files} ${_generated_external_srv_files}
-    install(
-      FILES ${_generated_srv_files}
-      DESTINATION "include/${PROJECT_NAME}/srv/dds_freertps"
-    )
-  endif()
-endif()
-
-else()
-
-  # generate specific type support code for the builtin_interfaces package
-  set(_generated_msg_files
-    ${rosidl_typesupport_freertps_cpp_TEMPLATE_DIR}/duration__type_support.cpp
-    ${rosidl_typesupport_freertps_cpp_TEMPLATE_DIR}/time__type_support.cpp
+  ament_export_include_directories(include)
+  install(
+    FILES ${_generated_msg_files}
+    DESTINATION "include/${PROJECT_NAME}/msg/freertps"
+  )
+  install(
+    FILES ${_generated_srv_files}
+    DESTINATION "include/${PROJECT_NAME}/srv/freertps"
```

Comment:
The export and install calls for the headers could be moved behind line 175 to the other export and install calls for the libraries. That makes it easier to spot what the CMake code exports and installs.